include/signal.h

```diff
@@ -386,7 +394,11 @@
  */
 
 #ifndef __SIGSET_T_DEFINED
-typedef uint32_t sigset_t;   /* Bit set of 32 signals */
+struct sigset_s
```

Comment:
Let's remove `sigisemptyset` or redirect to `nxsig_isemptyset`

Reply:
Done!  Good find, thanks!  This (along with sigorset and sigandset) were wrong anyway.